http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/resources/UrlOverrideResource.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.ts.http.restclient.reactive.resources;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@ApplicationScoped
```

Comment:
Kinda curious why you specify `@ApplicationScoped`, it is not wrong if there is a point and it cause no harm, but I don't get it from the context. Thanks